packages/kbn-event-annotation-components/components/annotation_editor_controls/tooltip_annotation_panel.tsx

```diff
@@ -6,16 +6,16 @@
  * Side Public License, v 1.
  */
 
-import { htmlIdGenerator, EuiFlexItem, EuiPanel, EuiText } from '@elastic/eui';
```

Comment:
Changes in this file turned out to be irrelevant to the bug, but I thought it was a nice simplification nonetheless.